roles/installer/molecule/default/verify.yml

```diff
@@ -1,14 +1,13 @@
 ---
-- name: Converge
+- name: Verify
   hosts: all
   gather_facts: false
-  vars:
-    foreman_installer_scenario: foreman
-    foreman_installer_verbose: true
-    foreman_installer_no_colors: false
-  roles:
-    - installer
   tasks:
+    - name: Foreman is installed
+      package:
+        name: foreman
+        state: present
```

Comment:
Maybe we should verify some artifact of the installer is present? Like https://github.com/theforeman/foreman-installer/blob/51d002636f178c0084483cb01f8aed7395a14900/hooks/boot/01-kafo-hook-extensions.rb#L11

Reply:
Or that we can access the web UI.

Reply:
I thought about the `.installed` file too, can add.

UI seems a bit overkill (and potentially slow in the container)

Reply:
added